tests/integration/tier0/check-user-response/test_user_response.py

```diff
@@ -2,8 +2,10 @@
 
 
 def test_check_user_response_user_and_password(convert2rhel):
-    # Run c2r registration with no username and password provided
-    # check for user prompt enforcing input, then continue with registration
+    """
+    Run c2r registration with no username and password provided
+    check for user prompt enforcing input, then continue with registration.
```

Comment:
```suggestion
    Run c2r registration with no username and password provided.
     Verify that user has to pass non empty username/password string to continue, otherwise enforce the input prompt again.
```